{{ cookiecutter.folder_name }}/docs/source/conf.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+import sphinx_rtd_theme
+module_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),'../../')
+sys.path.insert(0,module_path)
+
+# -- Project information -----------------------------------------------------
+
+project = '{{ cookiecutter.project_name }}'
+copyright = '2018, {{ cookiecutter.author_name }}'
+author = '{{ cookiecutter.author_name }}'
```

Comment:
I think this should just be `SLAC National Accelerator Laboratory`

Reply:
That's the default value for `cookiecutter.author_name`.